### ISSUE TITLE
Hide expired entries

### ISSUE THREAD

**Issue by daniel-mi** [PhilippC/keepass2android#1261]
It would be great if the expired entries did not appear in the search results. 

Optional: adjustable by configuration

**Comment by martyzigman** [comment]
I agree.  I like the app except for this one concern.

**Comment by MichiFr** [comment]
I've tons of entries in my password database, many are expired and striked out in any list, even in search results. 
But it is getting really annoying that this app still cannot hide expired entries.

Is this a feature hard to implement? Guess not, because insted of striking out one could simply hide them, right?

**Comment by Edwin-VC** [comment]
I totally agree that this would a must-have feature. The number of expired entries are accumulating and hence finding the correct entries does become harder. This should work without me having to manually update entries. Since the Icon is already showing the expired entries - thus expired entries are already recognized - I would expect this to be a low impact change. I could be wrong of course. But I'll donate you another beer 😉
